Spanner/src/Backup.php

```diff
@@ -186,6 +186,44 @@ public function create($database, \DateTimeInterface $expireTime, array $options
         return $this->resumeOperation($operation['name'], $operation);
     }
 
+    /**
+     * Create a copy of an existing backup in Cloud Spanner.
+     *
+     * Example:
+     * ```
+     * $spanner = new SpannerClient();
+     * $instance = $spanner->instance('destination-instance-id');
+     * $destBackup = $instance->backup('new-backup-id');
+     * $operation = $backup->createCopy($destBackup, new \DateTime('+7 hours'));
+     * ```
+     *
+     * @param string $newBackup The backup object that needs to be created as a copy.
+     * @param \DateTimeInterface $expireTime ​The expiration time of the backup,
+     *        with microseconds granularity that must be at least 6 hours and
+     *        at most 366 days. Once the expireTime has passed, the backup is
+     *        eligible to be automatically deleted by Cloud Spanner.
+     * @param array $options [optional] {
+     *         Configuration Options.
+     *
+     *         @type \DateTimeInterface $versionTime The version time for the externally
+     *              consistent copy of the database. If not present, it will be the same
+     *              as the create time of the backup.
+     *     }
+     * @return LongRunningOperation<Backup>
+     * @throws \InvalidArgumentException
+     */
+    public function createCopy($newBackup, $expireTime, array $options = [])
```

Comment:
`$newBackup` is an instance of `Backup`, so it should be typehinted. The same is true for `$expireTime`. 
```suggestion
    public function createCopy(Backup $newBackup, \DateTimeInterface $expireTime, array $options = [])
```